notifier/src/lib.rs

```diff
@@ -92,9 +93,21 @@ enum NotificationType {
     Log(Level),
 }
 
+#[derive(Clone)]
+pub enum NotificationType {
+    Trigger(String),
+    Resolve(String),
```

Comment:
How about this:
```suggestion
    Trigger { incident: Hash },
    Resolve { incident: Hash },
```
which:
1. self-documents the purpose of that `String`
2. delays the conversion from `Hash` to `String` until it's actually needed (ie, if/when the json message to PagerDuty is constructed)